authappliance/menu.py

```diff
@@ -541,13 +542,17 @@ def setup_redundancy(self):
 
         self.add_info("Setup my.cnf on local server...")
 
-        for key, value in shared_my_cnf_values.items() + local_my_cnf_values.items():
+        conf_values = shared_my_cnf_values.copy()
+        conf_values.update(local_my_cnf_values)
+        for key, value in conf_values:
```

Comment:
As we iterate over (key, value) pairs, I think this should be ``conf_values.items()`` :)

Reply:
You are right, missed that. Will fix.

Reply:
fixed in 8f45145